hype-submitter/src/test/java/com/spotify/hype/runner/KubernetesDockerRunnerTest.java

```diff
@@ -76,17 +76,8 @@ public void setUp() throws Exception {
   }
 
   @Test
-  public void setsImage() throws Exception {
-    RunEnvironment env = environment("busybox:1");
-    Pod pod = createPod(env);
-
-    Container container = findHypeRunContainer(pod);
-    assertThat(container.getImage(), is("busybox:1"));
-  }
```

Comment:
I'm not sure, the environment cannot set the image anymore. Do you mean I should test that subclassing a given HFn does indeed change the image?